test/e2e-secondary-network/secondary_network_test.go

```diff
@@ -449,6 +477,9 @@ func testSecondaryNetwork(t *testing.T, networkType string, pods []*testPodInfo)
 	if err := testData.assertPodNetworkStatus(t, clientset, pods, ns); err != nil {
 		t.Fatalf("Error when checking the Pod annotation: %v", err)
 	}
+	t.Run("testVerifyPodInterfaceMAC", func(t *testing.T) {
+		testData.verifyPodInterfaceMAC(t)
+	})
```

Comment:
why are we making this a dedicated subtest? It seems like we should have `testData.verifyMACAddresses` even before `pingBetweenInterfaces`.

Reply:
i added one check inside  ``` pingBetweenInterfaces```  function to verify mac address.